deps/leader_cron-0.1/rebar.config

```diff
@@ -0,0 +1,9 @@
+{erl_opts, [debug_info, fail_on_warning]}.
+{deps, [{gen_leader, ".*",
+	 {git, "git://github.com/jeraymond/gen_leader_revival.git",
```

Comment:
If this is indeed a dependency of `leader_cron` then it should be part of this PR.
Deps should never be downloaded at compile time.